examples/005_wakefields/t000_for_comparison.py

```diff
@@ -0,0 +1,382 @@
+import matplotlib.pyplot as plt
```

Comment:
remove all plots from the tests. We typically run the tests without installing matplotlib